docs/user/02-10-usage.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Use BTP Manager to manage SAP BTP Service Operator 
+---
+
+## Install SAP BTP Service Operator
```

Comment:
1) Remove content of the installation chapter, just place ref to main readme installation
2) New chapter between installation and usage, called "prepare and install secret", here use what we have about dummy and real secret
3) Usage: add a note, that usage make sense only with real cerdential, then all points will work. Mention that auditlog-api is a real service used as example